site.yml

```diff
@@ -14,6 +14,18 @@
   - { role: base }
   - { role: serf }
   - { role: dev }
+  - { role: vagrant }
+
+- hosts: build
```

Comment:
here is the gist that I think will fix `dev` role for us: https://github.com/mapuri/ansible/commit/4014dd94314afc87c9299544bf559657d07ba866